.github/workflows/basic-tests.yml

```diff
@@ -394,3 +394,57 @@ jobs:
         run: |
           breeze release-management generate-issue-content-core \
             --limit-pr-count 2 --previous-release 3.0.1 --current-release 3.0.2 --verbose
+
+  test-airflow-standalone:
+    timeout-minutes: 30
```

Comment:
If we wait for 10mins below, do we really need a 30minute timeout for the entire action?